src/test/java/org/xstefank/api/GitHubAPITest.java

```diff
@@ -0,0 +1,35 @@
+package org.xstefank.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.xstefank.model.Utils;
+
+import static org.mockito.Matchers.anyString;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({GitHubAPI.class, Utils.class})
+public class GitHubAPITest {
+
+    @Before
+    public void before() throws Exception {
+        PowerMockito.mockStatic(Utils.class);
+        PowerMockito.when(Utils.readTokenFromProperties(anyString(), anyString())).thenReturn(null);
```

Comment:
maybe try out also a valid case here?

Reply:
This is comment from previous upload but please check the https://github.com/xstefank/tyr/pull/25/files#diff-b32b0ffaf42b434a84013589995a8f08R31